### ISSUE TITLE
libprotobuf ERROR while running sentiment demo

### ISSUE THREAD

**Issue by cactiball** [baidu/Paddle#159]
1. paddle version
git version: commit 29c16e22420b8b69d17572accd2f6fc0239c8c7e
paddle compiled from source, with the options: cmake .. -DWITH_GPU=ON -DWITH_DOC=ON -DWITH_SWIG_PY=ON -DCMAKE_INSTALL_PREFIX=/data/user/program/pddl_build/
make and make install success.

2. protoc --version
libprotoc 2.5.0

3. run the sentiment demo:
cd /demo/sentiment/data/
./get_imdb.sh
cd ..
./preprocess.sh
the above steps are good.
However, run ./train.sh, I get the following error msg:

===================================================================
I1004 16:22:13.414708 12100 Util.cpp:144] commandline: /usr/local/bin/../opt/paddle/bin/paddle_trainer --config=trainer_config.py --save_dir=./model_output --job=train --use_gpu=false --trainer_count=4 --num_passes=10 --log_period=10 --dot_period=20 --show_parameter_stats_period=100 --test_all_data_in_one_period=1 
I1004 16:22:13.414949 12100 Util.cpp:113] Calling runInitFunctions
I1004 16:22:13.415261 12100 Util.cpp:126] Call runInitFunctions done.
[INFO 2016-10-04 16:22:13,802 networks.py:1125] The input order is [word, label]
[INFO 2016-10-04 16:22:13,802 networks.py:1132] The output order is [__cost_0__]
[libprotobuf ERROR google/protobuf/message_lite.cc:123] Can't parse message of type "paddle.TrainerConfig" because it is missing required fields: model_config.parameters[0].learning_rate, model_config.parameters[0].momentum, model_config.parameters[1].learning_rate, model_config.parameters[1].momentum, model_config.parameters[2].learning_rate, model_config.parameters[2].momentum, model_config.parameters[3].learning_rate, model_config.parameters[3].momentum, model_config.parameters[4].learning_rate, model_config.parameters[4].momentum, model_config.parameters[5].momentum, model_config.parameters[6].momentum, model_config.parameters[7].learning_rate, model_config.parameters[7].momentum, model_config.parameters[8].learning_rate, model_config.parameters[8].momentum, model_config.parameters[9].learning_rate, model_config.parameters[9].momentum, model_config.parameters[10].momentum, model_config.parameters[11].momentum, model_config.parameters[12].learning_rate, model_config.parameters[12].momentum, model_config.parameters[13].learning_rate, model_config.parameters[13].momentum, model_config.parameters[14].learning_rate, model_config.parameters[14].momentum, model_config.parameters[15].momentum, model_config.parameters[16].momentum, model_config.parameters[17].learning_rate, model_config.parameters[17].momentum
F1004 16:22:13.825393 12100 TrainerConfigHelper.cpp:59] Check failed: m->conf.ParseFromString(configProtoStr) 
*** Check failure stack trace: ***
    @     0x7fc3d0879daa  (unknown)
    @     0x7fc3d0879ce4  (unknown)
    @     0x7fc3d08796e6  (unknown)
    @     0x7fc3d087c687  (unknown)
    @           0x6aa559  paddle::TrainerConfigHelper::TrainerConfigHelper()
    @           0x6aa924  paddle::TrainerConfigHelper::createFromFlags()
    @           0x53af53  main
    @     0x7fc3cfa85ec5  (unknown)
    @           0x546695  (unknown)
    @              (nil)  (unknown)

/usr/local/bin/paddle: line 46: 12100 Aborted                 (core dumped) ${DEBUGGER} $MYDIR/../opt/paddle/bin/paddle_trainer ${@:2}
===================================================================

Could you please help to check what is this about?

**Comment by emailweixu** [comment]
It looks like your conflict version of paddle c++ binary and python interface. Your install prefix is different from the path shown in the above log

**Comment by cactiball** [comment]
Thank you very much! Append the install prefix to PATH in ~/.bashrc solved this issue.
